packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -1502,13 +1502,21 @@ export default abstract class SqlIntegration
           anonymousIdColumn: "user_pseudo_id",
           getMetricTableName: () => this.generateTablePath("events_*"),
           getDateLimitClause: (start: Date, end: Date) =>
-            `_TABLE_SUFFIX BETWEEN '${formatDate(
+            `((_TABLE_SUFFIX BETWEEN '${formatDate(
               start,
               "yyyyMMdd"
-            )}' AND'${formatDate(end, "yyyyMMdd")}'`,
+            )}' AND '${formatDate(
+              end,
+              "yyyyMMdd"
+            )}') OR (_TABLE_SUFFIX BETWEEN 'intraday_${formatDate(
+              start,
+              "yyyyMMdd"
+            )}' AND 'intraday_${formatDate(end, "yyyyMMdd")}'))`,
           getAdditionalEvents: () => [],
           getMetricWhereClause: (eventName: string) =>
-            `WHERE event_name = '${eventName}'`,
+            `((_TABLE_SUFFIX BETWEEN '{{date startDateISO "yyyyMMdd"}}' AND '{{date endDateISO "yyyyMMdd"}}') OR
+ (_TABLE_SUFFIX BETWEEN 'intraday_{{date startDateISO "yyyyMMdd"}}' AND 'intraday_{{date endDateISO "yyyyMMdd"}}')) 
+AND event_name = '${eventName}'`,
```

Comment:
This looks like it will break. This function is expecting to return a string that starts with WHERE. And I think it auto-appends the `getDateLimitClause` already if I'm not mistaken. I would revert this function entirely.

Reply:
Whoops.  I'll add the WHERE back in.  And no it doesn't append the getDateLimitClause.  It's what https://github.com/growthbook/growthbook/pull/1682 put in.